music21/musicxml/m21ToXml.py

```diff
@@ -3472,7 +3521,23 @@ def noteToXml(self, n: note.GeneralNote, noteIndexInChord=0, chordParent=None):
             for t in mxTieList:
                 mxNote.append(t)
 
-        # TODO: instrument
+        # instrument tags are necessary when there is more than one
```

Comment:
when there is more than one -- on the same Note?  (That's a musicxml 4.0 addition) or more than one on the same musicxml Part (or music21 part?)

Reply:
More than one anywhere on the musicxml `<part>`